manifests/init.pp

```diff
@@ -158,6 +179,13 @@
 #
 class consul (
   Hash $acls                                 = $consul::params::acls,
+  Hash $tokens                               = $consul::params::tokens,
+  Hash $policies                             = $consul::params::policies,
+  String $acl_api_hostname                   = $consul::params::acl_api_hostname,
+  String $acl_api_protocol                   = $consul::params::acl_api_protocol,
+  Integer $acl_api_port                      = $consul::params::acl_api_port,
```

Comment:
Probably this type can be used https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port.pp ?
```suggestion
  Stdlib::Port $acl_api_port                 = $consul::params::acl_api_port,
```

Reply:
Good idea from my point of view.
As far as I know this type was added in version 4.25 of stdlib. Is it ok to adjust the [current dependencies](https://github.com/solarkennedy/puppet-consul/blob/master/metadata.json#L13) accordingly?